docs/sdks/python/index.md

```diff
@@ -1 +1,315 @@
+​
+```python
+from algosdk import account, mnemonic
+
+def generate_algorand_keypair():
+    private_key, address = account.generate_account()
+    print("My address: {}".format(address))
+    print("My private key: {}".format(private_key))
+    print("My passphrase: {}".format(mnemonic.from_private_key(private_key)))
+```
+[`Watch Video`](https://youtu.be/ku2hFalMWmA?t=161)  
+[`More Information`](https://developer.algorand.org/docs/features/accounts/create/#standalone)
+
+!!! Tip
+    Make sure to save your account's address and passphrase at a seperate place, as they will be used later on.
+​
+!!! Warning
+    Never share Mnemonic and private key. Production environments require stringent private key management. For more information on key management in community Wallets, click [here](https://developer.algorand.org/docs/community/#wallets). For the [Algorand open source wallet](https://developer.algorand.org/articles/algorand-wallet-now-open-source/), click [here](https://github.com/algorand/algorand-wallet).
+
+# Fund the Account
+Before sending transactions to the Algorand network, the account must be funded to cover the minimal transaction fees that exist on Algorand. To fund the account use the [Algorand faucet](https://dispenser.testnet.aws.algodev.network/). 
+​
+[`Watch Video`](https://youtu.be/ku2hFalMWmA?t=257)  
+​
+!!! Info
+    All Algorand accounts require a minimum balance to be registered in the ledger. To read more about Algorand minimum balance, see [Account Overview](https://developer.algorand.org/docs/features/accounts/#minimum-balance).  
+
+# Connect Your Client
+Client must be instantiated prior to making calls to the API endpoints. You must provide values for `<algod-address>` and `<algod-token>`. The CLI tools implement the client natively. Code beyond this point will be put into the ***first_transaction_example*** function to create a single script.
+
+```python
+from algosdk.v2client import algod
+
+def first_transaction_example(private_key, my_address):
+    algod_address = "http://localhost:4001"
+    algod_token = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+    algod_client = algod.AlgodClient(algod_token, algod_address)
+```
+[`Watch Video`](https://youtu.be/ku2hFalMWmA?t=322)  
+
+!!! Info
+    The example code connects to the sandbox Algod client. If you want to connect to a Purestake client, see Purestake's [code samples](https://developer.purestake.io/code-samples).
+
+# Check Your Balance
+Before moving on to the next step, make sure your account has been funded by the faucet.
+
+```python
+    account_info = algod_client.account_info(my_address)
+    print("Account balance: {} microAlgos".format(account_info.get('amount')) + "\n")
+```
+[`Watch Video`](https://youtu.be/ku2hFalMWmA?t=381) 
+
+# Build First Transaction
+Communication with the Algorand network is performed using transactions. To create a payment transaction use the following code.
```

Comment:
Maybe: Use transactions to communicate with the Algorand network.

Reply:
consider replacing communicate with interact to emphasize an effect?